main.js

```diff
@@ -0,0 +1,87 @@
+const { app, BrowserWindow, globalShortcut, ipcMain} = require('electron');
+const path = require('path');
+const rootPath = require('electron-root-path').rootPath;
+const log = require('electron-log');
+require('./src/menu.js');
+const {getLogTransportConsole, getLogTransportFile, getLogResolvePath} = require('./src/lib/logFormat');
+const ipc = ipcMain;
+const ConstructURL = require('./src/utils/ConstructURL');
+
+function createWindow () {
+    // Method to format the writings logs
+    getLogTransportConsole();
+    getLogTransportFile();
+    getLogResolvePath();
+
+    // Create main window
+    let mainWindow = new BrowserWindow({
+        width: 800,
+        height: 600,
+        icon: getAppIcon(),
+        webPreferences: {
+            nodeIntegration: true,
+            contextIsolation: false,
+            preload: path.join(__dirname, './preload.js')
+        }
+    });
+
+    // Allow you to open devtools
+    globalShortcut.register('CommandOrControl+I', () => { mainWindow.webContents.openDevTools(); });
+
+    // if is dev mode passe argument to true
```

Comment:
mix french and english ;)
and the next lines are not working as describing by the comment